modules/nw-ingresscontroller-change-external.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+[id="nw-ingresscontroller-change-external_{context}"]
+= Configuring the Ingress Controller endpoint publishing scope to External
+
+When a cluster administrator installs a new cluster without specifying that the cluster is private, the default Ingress Controller is created with a `scope` set to `External`.
+
+The Ingress Controller's scope can be configured to be `Internal` during installation or after, but cluster administrators can change an `Internal` Ingress Controller to `External`.
+
+.Prerequisites
+
+* You installed the `oc` CLI.
+
+.Procedure
+
+* To change an `Internal` scoped Ingress Controller to `External`, enter the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator patch ingresscontrollers/private --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"type":"LoadBalancerService","loadBalancer":{"scope":"External"}}}}'
+----
++
+.Verification
++
+* To check the status of the Ingress Controller, enter the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress-operator get ingresscontrollers/default -o yaml
+----
++
+** The `Progressing` status condition indicates whether you must take further action. For example, the status condition can indicate that you need to delete the Service by entering the following command:
```

Comment:
This is an important detail, and maybe it should be called out earlier and more starkly:  On some platforms, it is necessary to delete and recreate the service.  More importantly, changing the scope may cause disruption to ingress traffic (potentially for several minutes), especially on platforms where it is necessary to delete and recreate the service, because the procedure may cause OpenShift to deprovision the existing service load-balancer, provision a new one, and update DNS.

Reply:
Added this as an [IMPORTANT] note prior to the procedure.